src/http/config.rs

```diff
@@ -28,11 +28,11 @@ pub fn default_server_config() -> AybConfig {
             smtp_port: 465,
             smtp_username: "login@example.org".to_string(),
             smtp_password: "the_password".to_string(),
-            templates: None,
         },
         cors: AybConfigCors {
             origin: "*".to_string(),
         },
+        web: None,
```

Comment:
is there an easy way to add coverage to the end-to-end test around this? low bar, let's create an issue to cover the web information functionality in end-to-end tests in the future.

Reply:
I checked if reqwest had support for data URIs, but that's not the case, so there isn't an easy way to E2E test this.
I can add the E2E tests in a follow-up PR, unless you want both things to land together :)

Reply:
I think creating an issue and doing it in a follow-up PR is just fine! Thank you! :)